Zend/tests/bug66218.phpt

```diff
@@ -3,7 +3,6 @@ Bug #66218 zend_register_functions breaks reflection
 --SKIPIF--
 <?php
 if (PHP_SAPI != "cli") die("skip CLI only test");
-if (!function_exists("dl")) die("skip need dl");
```

Comment:
Wasn't this one removed http://br2.php.net/dl?

Reply:
It's still available one some SAPIs, iirc cli and cgi.